### DISCUSSION TITLE
Reference XF for debugging in own app

### DISCUSSION THREAD

**Discussion by AlleSchonWeg** [xamarin/Xamarin.Forms#14884] [Q&A]
Hi,
i cloned this git repositoy and i'm able to run the Xamarin.Forms.ControlGallery for example. My question is, how can i reference this repository in my own XF App instead of the nuget package?
My goal is to debugging the XF code and make changes.
Thank you

**Comment by lee-m** [comment]
The approach I took when I had to fix something was to create a repro in the control gallery and then simply debug it from there rather than try and reference my locally build XF assemblies.
